opensearch/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     implementation group: 'org.json', name: 'json', version:'20230227'
     compileOnly group: 'org.opensearch.client', name: 'opensearch-rest-high-level-client', version: "${opensearch_version}"
     implementation group: 'org.opensearch', name:'opensearch-ml-client', version: "${opensearch_build}"
+    implementation group: 'org.opensearch', name: 'common-utils', version: "${opensearch_build}"
```

Comment:
Did this fix the error? 
Or it automatically got fixed after the latest changes of opensearch got pushed to mvn repo.

Reply:
I just restarted CI on `main` it failed: https://github.com/opensearch-project/sql/actions/runs/5320847765/jobs/9655483529. So fix is still required.

Reply:
Re-run the CI, only see the following errors. Does common-utils necessary?

```
Execution failed for task ':opensearch-sql-plugin:compileTestJava'.
AstExpressionBuilderTest > canBuildArithmeticExpressionPrecedence() PASSED
> Could not resolve all dependencies for configuration ':opensearch-sql-plugin:testCompileClasspath'.

   > Conflict(s) found for the following module(s):
AstExpressionBuilderTest > canBuildComparisonExpression() PASSED
       - org.mockito:mockito-core between versions 5.4.0 and 5.2.0
     Run with:
         --scan or
         :opensearch-sql-plugin:dependencyInsight --configuration testCompileClasspath --dependency org.mockito:mockito-core
     to get more insight on how to solve the conflict.
```

Reply:
Right, perhaps snapshots was updated which makes this dependency unneeded. Reverted in 66b98fd28.
Thank you for testing!